maven/plugins/common/src/main/java/com/vmware/pscoe/maven/plugins/AbstractIacMojo.java

```diff
@@ -181,14 +227,24 @@ protected ConfigurationCs getConfigurationForCs() throws ConfigurationException
 		}
 	}
 
+	/**
+	 * Retrieve vRO rest client for vRO interaction.
+	 * 
+	 * @return vRO rest client
+	 * @throws ConfigurationException
+	 */
 	protected RestClientVro getVroRestClient() throws ConfigurationException {
 		return RestClientFactory.getClientVro(getConfigurationForVro());
 	}
 
+	/**
+	 * Retrieve vRA ng rest client for vRA ng interaction.
+	 * 
+	 * @return vRA ng rest client
+	 * @throws ConfigurationException
+	 */
 	protected Optional<Configuration> getConfigurationForType(PackageType type) throws ConfigurationException {
-		if (PackageType.VRA == type) {
-			return Optional.ofNullable(ConfigurationVra.fromProperties(getConfigurationProperties(type, vra, "vra.")));
-		} else if (PackageType.VRO == type) {
+		if (PackageType.VRO == type) {
```

Comment:
Interesting, I just noticed here we don't have vra ng and vrops (even before the pr) - is this related to optional push options like vro.packageImportConfigurationAttributeValues?

Reply:
We do have `VRANG` and `VROPS` 😄

Reply:
Yeap, it was just a few lines hidden and I didn't see the extend button, I just double checked the source and when I saw it there, I saw the extend button here as well :D